sky/packages/sky/lib/src/rendering/binding.dart

```diff
@@ -92,10 +92,12 @@ class _UiEventConverter {
         break;
     }
 
+    int pointer = (state == null) ? event.pointer : state.pointer;
```

Comment:
Personally I find it easier to read with ( ) because of the visual confusion that x = y == z ... might cause, but I'm happy to use local style if it's preferred without